package.json

```diff
@@ -50,11 +50,11 @@
     "json-date-parser": "^1.0.1",
     "ts-jest": "^27.0.3",
     "tslint": "^6.1.3",
-    "typescript": "^4.3.4",
-    "validator": "^13.6.0"
```

Comment:
`validator` is referenced by non-dev/test code and therefore must remain as a dependency.

Reply:
I've moved `validator` to the `dependencies` list instead of `devDependencies`, are you ok with that?

Reply:
Yes. That's fine thanks. Strange that it had been missed to start with.